### PR TITLE
MYB-453: Added empty screen for favs' table and more

### PR DESCRIPTION
**Pull request by lisandro**
- **MYB-453 MYB-452**: Added DZNEmptyDataSet to display an image with text when favs' datasource is empty
img_0401
img_0400

- **MYB-438:** Recent won't be saved if coordinates already exists OR address string also
- Fixed label issue with iPhone 5S resolution
**Before**
img_0399
**After**
img_0402

### CODE REVIEW

**Review by marcosjvivar — APPROVED** [review]
Changes are OK and ready to be merged! 🚀 and :shipit:

File: MyBus/MyBus/FavoriteViewController.swift [hunk 1]
```diff
@@ -87,5 +93,26 @@ class FavoriteViewController: UIViewController, UITableViewDelegate
...
+    func titleForEmptyDataSet(scrollView: UIScrollView!) -> NSAttributedString! {
+        return NSAttributedString(string: "No hay favoritos")
+    }
+    
+    func descriptionForEmptyDataSet(scrollView: UIScrollView!) -> NSAttributedString! {
+        let attributes: [String : AnyObject] = [
+            NSForegroundColorAttributeName: UIColor.lightGrayColor()
+        ]
+        return NSAttributedString(string: "Agregá tu primer ubicación favorita para buscar más rápido", attributes: attributes)
+    }
```
Comment (ignaciodevspark) [inline MyBus/MyBus/FavoriteViewController.swift:110]: We should add this string to the localized file

**Comment by ignaciodevspark** [comment]
Added a small comments. Apart from that looks good to merge!

**Comment by safink** [comment]
Excellent work! Approved 👍

**Pull request merged**